lib/matplotlib/font_manager.py

```diff
@@ -201,7 +201,7 @@ def win32InstalledFonts(directory=None, fontext='ttf'):
                     # is fixed in Py>=3.6.1.
                     direc = direc.split("\0", 1)[0]
                     path = Path(directory, direc).resolve()
-                    if path.suffix.lower() in fontext:
```

Comment:
seems easier to prepend a dot to fontext at line 191 (as in list_fonts)?

Reply:
Indeed

Reply:
`fontext` can be either tuple or tuple of strings, it will require something like

```python
if isinstance(fontext, str):
    fontext = '.' + fontext
else:
    fontext = ['.' + ext for ext in fontext]
```

Should I do it?

Reply:
You need to make it a list of strings so that we don't get false positives with extensionless files (`"" not in [".ttf"]` but `"" in ".ttf"`)

Thanks for investigating.

Reply:
Right, will fix it.